src/lib/components/Pagination/Pagination.types.ts

```diff
@@ -0,0 +1,21 @@
+import { ComponentProps, PropsWithChildren, ReactEventHandler, ReactNode } from 'react';
+
+export type PaginationProps = PropsWithChildren<Pagination>;
```

Comment:
Could you please explain why did you created this `.types.ts` file just for this component? What is the goal here? I normally prefer to have all the types and interfaces being created/defined where we are using it,

Reply:
Initially it was to avoid importing type A in B file and B in A file.